openbao.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: openbao
+  version: 2.0.2
+  epoch: 0
+  description: OpenBao exists to provide a software solution to manage, store, and distribute sensitive data including secrets, certificates, and keys.
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+      - yarn
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/openbao/openbao
+      tag: v${{package.version}}
+      expected-commit: 96853bb4de27ab8ffd1b0c2898c691460d43edeb
+
+  - runs: |
+      # UI not embedded until it is changed ; see https://github.com/openbao/openbao/issues/129
+      # TODO: re-enable when a release with new UI is done and add "ui" to tags below
+      # make ember-dist
+
+  - uses: go/build
+    with:
+      # TODO: add ui to tags when building it above
+      tags: "openbao"
+      packages: .
+      output: bao
+      ldflags: |
+        -X github.com/openbao/openbao/version.fullVersion=${{package.version}}
+        -X github.com/openbao/openbao/version.GitCommit='$(git rev-parse HEAD)'
+        -X github.com/openbao/openbao/version.BuildDate='$(git show --no-show-signature -s --format=%cd --date=format:"%Y-%m-%dT%H:%M:%SZ" HEAD)'
+
+  - uses: strip
+
+subpackages:
+  - name: openbao-compat
+    description: "Compatibility package to allow replacing vault image"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr/bin
```

Comment:
Since we aren't linking anything here, we can remove this line

Reply:
Good point. Done.